### PR TITLE
Index topics (Old)

### PR DESCRIPTION
**Pull request by sjtg**
### Change UI design for Index Topics Section. Below are UI Changes:
1: Navbar - New Edulution Log
2: Table   - Modified Index Topic Table
3: Pagination  - Added Pagination to the Index Topics 
4: Search -

##### 1: Index Topics 
Before_1

#####  1: Index Topics
After_1

…

### Reviewer guidance

…

### References

…

----

### Contributor Checklist

PR process:

- [x] PR has the correct target branch
- [x] PR has 'needs review' label
- [x] If PR is ready for review, a reviewer has been added. (Don't use 'Assignees')
- [ ] If this is an important user-facing change, PR or related issue has a 'changelog' label
- [ ] If this includes an internal dependency change, a link to the diff is provided

Testing:

- [ ] Contributor has fully tested the PR manually
- [ ] If there are any front-end changes, before/after screenshots are included

### Reviewer Checklist

- PR is fully functional
- PR has been tested for [accessibility regressions]
- External dependency files were updated if necessary (node_modules, libraries, etc)
- Documentation is updated

### CODE REVIEW

File: portal/playlists/numeracy/alpha/alpha_a.html [hunk 1]
```diff
@@ -1,8 +1,60 @@
 <!DOCTYPE html>
-<html ng-app="offlineTests">
+<html ng-app="numeracyDashoard">
 
 <head>
     <title>Alpha A</title>
+<<<<<<< HEAD
```
Comment (Aypak) [inline portal/playlists/numeracy/alpha/alpha_a.html:6]: Merge conflict!

File: portal/playlists/numeracy/alpha/alpha_a.html [hunk 2]
```diff
@@ -1,8 +1,60 @@
 <!DOCTYPE html>
-<html ng-app="offlineTests">
+<html ng-app="numeracyDashoard">
```
Comment (Aypak) [inline portal/playlists/numeracy/alpha/alpha_a.html:2]: The word Dashboard is misspelled!

**Comment by Aypak** [comment]
Feedback on the above:
1. The name of the page and the heading is **Index of Topics**, not Index Topic.
2. The logo is too small and not vertically centered. Kindly use a bigger logo which takes at least 40-60% of the navbar width, ensure that it is centered vertically and horizontally.
3. The text and arrow in the back button are not vertically centered.
4. Kindly use the same font family as the original page.
5. This PR contains numerous changes to other files. Some even contain unresolved merge conflicts, and other changes that have not yet been reviewed. Kindly review your work and make a few changes at a time.

**Comment by sjtg** [comment]
Hi Kapya I have made changes to the portal based on your feedback:
I have upload new screenshot.
1.   The name of the page and the heading is Index of Topics
2.     The logo navbar increased in size  centered vertically and horizontally.
3.     The text and arrow in the back button are not vertically centered.
4.     Original font family page.
5.     Merge conflicts resolved.

I will await for your feedback.

**Pull request closed without merging**